packages/@webex/media-helpers/package.json

```diff
@@ -17,7 +17,7 @@
     "node": ">=16"
   },
   "dependencies": {
-    "@webex/internal-media-core": "2.2.9",
+    "@webex/internal-media-core": "link:../../../../webrtc-media-core",
```

Comment:
We should remove this

Reply:
Yep, this and the package.json file in plugin-meetings will be changed once this PR is finalized. (We didn't want to make this change until the very end to prevent it from blocking other changes in internal-media-core.)